docs/HACKING.md

```diff
@@ -40,10 +48,15 @@ hack/olm.sh cleanup -c "<OPERATOR_IMAGE>"
 ```
 
 ### Running e2e tests on a cluster
-We need to set up all the environment variables required in [Development workflow](#development-workflow) as well as: 
+The following enviroment variables must be set:
 ```shell script
+# SSH key to be used for creation of cloud-private-key secret
+export KUBE_SSH_KEY_PATH=<path to RSA type ssh key>
 export OPERATOR_IMAGE=<registry url for remote WMCO image>
 export OPENSHIFT_CI=false
+export WMCO_VERSION=<current WMCO semver, i.e 2.0.0>
```

Comment:
What happens if I set a version that is different from the yaml file?

Reply:
`operator-sdk run packagemanifests` will not work

Reply:
Added documentation in the manifests creation portion